llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -753,6 +755,48 @@ class SelectInst : public Instruction {
 #endif
 };
 
+class InsertElementInst final : public Instruction {
+  /// Use Context::createInsertElementInst(). Don't call
+  /// the constructor directly.
```

Comment:
I think we can drop this comment (or make it fit in a single line).

Reply:
Reworded it into a single line.